app/javascript/src/components/Dashboard/Notes/Pane/Form/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui";
+import { Input, Textarea, Select } from "neetoui/formik";
+import { useTranslation } from "react-i18next";
+
+import notesApi from "apis/notes";
+
+import { TAGS, VALIDATION_SCHEMA } from "./constants";
+
+const Form = ({ onClose, refetch, note, isEdit }) => {
+  const { t } = useTranslation();
+
+  const handleSubmit = async values => {
+    try {
+      if (isEdit) {
+        await notesApi.update(note.id, values);
+      } else {
+        await notesApi.create(values);
+      }
+      refetch();
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={note}
+      validationSchema={VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ isSubmitting }) => (
+        <FormikForm className="w-full">
```

Comment:
Pass in `noValidate` here to disable [validation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes) from the HTML form element.